ng-dev/pr/common/validation/validations.ts

```diff
@@ -6,13 +6,15 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {Commit} from '../../commit-message/parse';
-import {TargetLabel, TargetLabelName} from './target-label';
-import {MergeConfig} from '../config';
+import {Commit} from '../../../commit-message/parse';
+import {TargetLabel, TargetLabelName} from '../targeting/target-label';
+import {MergeConfig} from '../../config';
 import {PullRequestFailure} from './failures';
-import {red, warn} from '../../utils/console';
-import {breakingChangeLabel} from './constants';
-import {RawPullRequest} from '../common/fetch-pull-request';
+import {red, warn} from '../../../utils/console';
+import {RawPullRequest} from '../fetch-pull-request';
+
+/** Label for pull requests containing a breaking change. */
+export const breakingChangeLabel = 'flag: breaking change';
```

Comment:
This seems wrong to have here to be honest. There should be one canonical place where this is hard-coded and it should be in a easy-to-discover file that is not concerned with something else, like validation IMO

Reply:
Agreed, I will see where I can find that is a better option